src/components/cards/CardMenu.vue

```diff
@@ -129,6 +129,9 @@ export default {
 		isCurrentUserAssigned() {
 			return this.card.assignedUsers.find((item) => item.type === 0 && item.participant.uid === getCurrentUser()?.uid)
 		},
+		activeBoards() {
+			return this.$store.getters.boards.filter((item) => item.deletedAt === 0)
```

Comment:
For additional improvement, it could also filter out archived boards here:

```suggestion
			return this.$store.getters.boards.filter((item) => item.deletedAt === 0 && item.archived === false)
```